### ISSUE TITLE
speech command related issue on custom dataset

### ISSUE THREAD

**Issue by dimanshu** [tensorflow/tensorflow#40142]
steps i followed for making dataset
1) collect 4 word (number,tollfree ,call,virtual) 
convert them into TTS and changed its pitch to make more data and achieved 
2500 data per sample
2) while training im getting an accuracy of 100% and validation is also 100% but when i test this on call data the result are all wrong.
99% is all wrong.
`I0604 07:34:51.632841 140415957108544 train.py:242] Step #1500: rate 0.000020, accuracy 100.00%, cross entropy 0.004794`

i dont know what im doing wrong

**Comment by Saduf2019** [comment]
@dimanshu 
This doesnt seem like a tensorflow issue,  In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here

**Comment by Saduf2019** [comment]
@dimanshu
Please update as per above comment.
